components/Card/index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Card</title>
+    <link rel="stylesheet" href="/components/sideNav/sideNav.css">
+    <link rel="stylesheet" href="card.css">
+    <link rel="icon" type="image/png" href="https://monkey-group.be/wp-content/uploads/2020/11/Favicon-2.png"/>
+</head>
+<body>
+    <nav class='navigation-bar' >
+        <div class='list-parent-left' id='symbol'>
+            <a class='li-link'href='../../index.html'> <h3>Chikku<span class='heading-color'>Ui</span></h3> </a>  
+        </div>
+        <div class='list-parent-right' id='modal'>
+            <a  href="javascript:void(0);" class="li-icon" onclick="modelFunction()">
+                <i class="fa fa-bars"></i>
+              </a>
+              <a class='li-link' href='../../index.html'> Home</a>
```

Comment:
If you are naming the class `li-link`, then why not use `<ul>` `<li>` tags, for displaying the items.